package.json

```diff
@@ -24,24 +24,29 @@
   "author": "DesignrKnight",
   "license": "ISC",
   "bugs": {
-    "url": "https://github.com/dscnitrourkela/project-guava/issues"
+    "url": "https://github.com/riteshsp2000/project-guava/issues"
   },
-  "homepage": "https://github.com/dscnitrourkela/project-guava#readme",
+  "homepage": "https://github.com/riteshsp2000/project-guava#readme",
   "dependencies": {
+    "apollo-server-express": "^2.19.2",
+    "cookie-parser": "^1.4.5",
+    "cors": "^2.8.5",
+    "csurf": "^1.11.0",
+    "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "express-graphql": "^0.11.0",
-    "graphql": "^15.3.0"
+    "firebase-admin": "^9.4.2",
+    "graphql": "^15.4.0",
+    "mongodb": "^3.6.3",
+    "mongoose": "^5.11.12"
   },
   "devDependencies": {
-    "chai": "^4.2.0",
-    "eslint": "^7.14.0",
+    "colors": "^1.4.0",
```

Comment:
Why colours in devDependencies?

Reply:
colors is used for console logging. And will be used only in development. So a dev dependency